Refactor(eos_designs)!: Remove wan_use_evpn_node_settings_for_lan and make it the default behavior (#5985)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

| `underlay_mutlticast` | `underlay_mutlticast_pim_sm` or `<node_type_key>.defaults.underlay_multicast.pim_sm.enabled` or `<node_type_key>.node_groups[].underlay_multicast.pim_sm.enabled` or `<node_type_key>.node_groups[].nodes[].underlay_multicast.pim_sm.enabled` or `<node_type_key>.nodes[].underlay_multicast.pim_sm.enabled` |
| `underlay_ospf_authentication.message_digest_keys[].key` | `underlay_ospf_authentication.message_digest_keys[].cleartext_key` |

### Removed the support  of `wan_use_evpn_node_settings_for_lan` key

In AVD 5.x, the key `wan_use_evpn_node_settings_for_lan` was used as preview to use `overlay_routing_protocol`, `evpn_role` and `vtep` node settings for LAN side on WAN devices

> Starting with AVD 6.0.0, this will be the default option will be removed.

Reviewer: ```suggestion
Starting with AVD 6.0.0, this is the only possible behavior and the key has been removed.
```

Reviewer: it also means that it is not possible anymore to exclude a VRF from the wan using Address families only

Reviewer 2: well if you just don't define it under the wan policies you are good right?

Author: updated
